std/node/util.ts

```diff
@@ -42,60 +42,74 @@ export function inspect(object: unknown, ...opts: any): string {
   });
 }
 
+/** Checks if a value is an array. */
```

Comment:
Some functions are deprecated in NodeJS, so write them down?

Reply:
Should I rewrite like below?
```
/** 
 * Checks if a value is an array. 
 * Deprecated in node.js. Use Array.isArray() instead.
 */
```